ci/cpu/build.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash -ex
+
+export PATH=/conda/bin:/usr/local/cuda/bin:$PATH:$WORKSPACE
+export HOME=$WORKSPACE
+
+BRANCH="$(git rev-parse --abbrev-ref HEAD)"
+if [[ "$BRANCH" != "main" ]]; then
+  echo 'Branch release not main, exiting...';
+  exit 1;
+fi
+
+cat <<EOF > credentials.sh
+#!/bin/bash
+echo username=$GIT_USERNAME
+echo password=$GIT_PASSWORD
+EOF
+
+git config --global user.email "gputester@rapids.ai"
+git config --global user.name "GPU Tester"
+git config --global credential.helper "/bin/bash $WORKSPACE/credentials.sh"
+
+source activate gdf
```

Comment:
```suggestion
source activate rapids
```

`gdf` is our old `conda` environment name. It's been replaced with `rapids`, but a symlink still remains to prevent breakage on old scripts ([src](https://github.com/rapidsai/gpuci-build-environment/blob/branch-0.19/rapidsai/base-runtime.Dockerfile#L67-L68)).